svc/WebSocketService.ts

```diff
@@ -173,7 +173,11 @@ export class WebSocketService extends HoistService {
     heartbeatOrReconnect() {
         this.updateConnectedStatus();
         if (this.connected) {
-            this.sendMessage({topic: this.HEARTBEAT_TOPIC, data: 'ping'});
+            this.sendMessage({
+                topic: this.HEARTBEAT_TOPIC,
+                data: 'ping',
+                clientAppVersion: XH.appVersion
```

Comment:
I initially made `data` an object and sent `clientAppVersion` as a property in said object.

I am unsure though how clients may be using this service. If they are extending and overriding `data` in `sendMessage` we would lose this value. However, there is also nothing stopping them from overriding `heartbeatOrReconnect` either.
Open to suggestions or notes to help better understand real world use cases.